executor/ddl.go

```diff
@@ -348,6 +348,13 @@ func (e *DDLExec) dropTableObject(objects []*ast.TableName, obt objectType, ifEx
 		}
 	}
 	if len(notExistTables) > 0 && !ifExists {
+		for _, table := range notExistTables {
+			if obt == sequenceObject {
+				e.ctx.GetSessionVars().StmtCtx.AppendError(infoschema.ErrSequenceDropExists.GenWithStackByArgs(table))
```

Comment:
I do not think it is a good idea to append the error here, when an error happens, it should be shown in `show errors` not only this error.

Reply:
I mean, all the errors. So it is better to append the error before it returns to the client

Reply:
do you mean append all errors in one error message?

Reply:
no, I mean something like https://github.com/pingcap/tidb/blob/dffba2195600883b62231f51676b7bbd7d3b5a17/session/session.go#L1294-L1297

then we can handle all the errors, not only the `unknown table`. 
such as 
```sql
mysql> create table t(a int);
ERROR 1050 (42S01): Table 't' already exists
mysql> show errors;
+-------+------+--------------------------+
| Level | Code | Message                  |
+-------+------+--------------------------+
| Error | 1050 | Table 't' already exists |
+-------+------+--------------------------+
1 row in set (0.00 sec)
```

Reply:
ok, I add `AppendError` in `func runStmt`, I'm not sure if this is the right place
PTAL @xiongjiwei

Reply:
Thanks! the function `session.ExecuteStmt` calls `runStmt` and the error is recorded in some function which calls `session.ExecuteStmt` (see above). it may lead to redundant error in `show errors`. I think only append the error in `runStmt` and remove others will be better.(I have not check it carefully)
BTW, can you please add some tests?

Reply:
ok, I will modify it.

Reply:
I have added some tests, PTAL @xiongjiwei